TypeCobol/Compiler/CodeModel/SymbolTable.cs

```diff
@@ -849,7 +857,7 @@ public IList<Paragraph> GetParagraph(SymbolReference symbolRef, Section sectionN
         /// <returns>The collection of paragraph names</returns>
         public IEnumerable<Paragraph> GetParagraphs(Func<Paragraph, bool> predicate)
```

Comment:
This method is also called in `OnDefinition`, so adding the sections there should make the 'Go To Definition' work for sections too.

Reply:
Looks like we never reach this part because of a bug in last significant token detection. Anyway, it seems betetr to add the sections here, as long as it does not break any of our LSR tests.

Reply:
Done